enterprise/cmd/executor/internal/run/install.go

```diff
@@ -291,19 +292,19 @@ func installCNIPlugins(cliCtx *cli.Context) error {
 	return nil
 }
 
-func installSrc(cliCtx *cli.Context, logger log.Logger, config *config.Config) error {
+func installSrc(runner util.CmdRunner, cliCtx *cli.Context, logger log.Logger, config *config.Config) error {
```

Comment:
same here, I think context should always be first (yes this is cliCtx but it also contains the ctx)

```suggestion
func installSrc(cliCtx *cli.Context, runner util.CmdRunner, logger log.Logger, config *config.Config) error {
```